### ISSUE TITLE
Abnormal behavior when access computed value and modify it at same reaction 

### ISSUE THREAD

**Issue by fedeoo** [mobxjs/mobx#916]
I have a Question: 
Chrome 56, Mobx version 3.1.7
```
const store = observable({
  a: 3,
  b: 4,
  get sum() {
    return this.a + this.b;
  },
});
autorun(() => {
  console.log('sum is', store.sum);
  store.b = 5;
});
store.a = 6;
```
The above code print the result only once. 

I doubt that there is a bug.

**Comment by andykog** [comment]
@fedeoo, nice catch! Working on it.

**Comment by mweststrate** [comment]
Released fix as `3.1.8`.
